### PR TITLE
Cover variable expansion in topic authorisation

### PR DESCRIPTION
**Pull request by acogoluegnes**
Part of rabbitmq/rabbitmq-server#1229

### CODE REVIEW

File: site/access-control.xml [hunk 1]
```diff
@@ -275,11 +275,16 @@ loopback_users = none
...
           href="man/rabbitmqctl.1.man.html#Access%20control">Access
           Control section</a> of the <a
           href="man/rabbitmqctl.1.man.html">rabbitmqctl(1) man
-          page</a>.
+          page</a>. Note the default authorisation backend supports variable expansion
+          in the permission regular expression.
+          Three variables are supported: <code>username</code>, <code>vhost</code>,
+          and <code>client_id</code> (only when using MQTT). For example, the permission
+          <code>^{username}-.*</code> would be expanded to <code>^tonyg-.*</code>
+          if <code>tonyg</code> was the connected user.
```
Comment (lukebakken) [inline site/access-control.xml]: This may make it clearer that those variables are MQTT-only:

----

When using the MQTT protocol, three variables are supported: `<code>username</code>`, `<code>vhost</code>`, and `<code>client_id</code>`. For example, when `<code>tonyg</code>` is the connected user, the permission `<code>^{username}-.*</code>` is expanded to `<code>^tonyg-.*</code>`.

Comment (lukebakken) [inline site/access-control.xml]: On my first reading, I thought _all_ three of those variables were limited to MQTT. Now that I know better, how about this wording?

----

Three variables are supported: `<code>username</code>`, `<code>vhost</code>`, and `<code>client_id</code>`. Note that `<code>client_id</code>` is expanded only when using MQTT. For example, if `<code>tonyg</code>` is the connected user, the permission `<code>^{username}-.*</code>` is expanded to `<code>^tonyg-.*</code>`.

Comment (acogoluegnes) [inline site/access-control.xml]: LGTM, I'll change it.
